<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>417</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>269</width>
    <height>417</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>269</width>
    <height>417</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
            background-color: #f8f9fa;
            color: #333333;
            font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, sans-serif;
            font-size: 14px;
        }

        /* 滚动条美化 */
        QScrollBar:vertical {
            background: #f0f0f0;
            width: 10px;
            margin: 0px;
            border-radius: 5px;
        }

        QScrollBar::handle:vertical {
            background: #c0c0c0;
            min-height: 30px;
            border-radius: 5px;
        }

        QScrollBar::add-line:vertical,
        QScrollBar::sub-line:vertical {
            height: 0px;
        }

        /* 按钮样式增强 */
        QPushButton {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #4a90e2, stop:1 #3a7bd5);
            color: white;
            border: none;
            padding: 8px 16px;
            border-radius: 4px;
            font-weight: 500;
            transition: all 0.2s;
        }

        QPushButton:hover {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #5a9ff2, stop:1 #4a8be5);
            transform: translateY(-1px);
            box-shadow: 0 2px 4px rgba(0,0,0,0.1);
        }

        QPushButton:pressed {
            background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #3a7bd5, stop:1 #2a6bc5);
            transform: translateY(0);
            box-shadow: none;
        }

        /* 文本输入框增强 */
        QLineEdit {
            border: 1px solid #e0e0e0;
            border-radius: 4px;
            padding: 6px 10px;
            background: white;
            transition: border-color 0.2s;
        }

        QLineEdit:focus {
            border-color: #4a90e2;
            box-shadow: 0 0 0 2px rgba(74, 144, 226, 0.2);
            outline: none;
        }

        /* 表格样式优化 */
        QTableView {
            gridline-color: #e8e8e8;
            background-color: white;
            border: 1px solid #e0e0e0;
            border-radius: 4px;
            selection-background-color: #e6f7ff;
            selection-color: #333333;
        }

        QTableView::item {
            padding: 6px 8px;
            border-bottom: 1px solid #f0f0f0;
        }

        QTableView::item:selected {
            background-color: #d6eaf8;
        }

        QHeaderView::section {
            background-color: #f5f7fa;
            color: #666666;
            padding: 6px 8px;
            border-right: 1px solid #e0e0e0;
            border-bottom: 1px solid #e0e0e0;
            font-weight: 500;
        }

        QHeaderView::section:last {
            border-right: none;
        }

        /* 标签样式优化 */
        QLabel {
            color: #333333;
            font-size: 14px;
        }

        QLabel[class=&quot;Title&quot;] {
            font-size: 18px;
            font-weight: 600;
            color: #222222;
        }

        QLabel[class=&quot;Subtitle&quot;] {
            font-size: 16px;
            font-weight: 500;
            color: #444444;
        }

        /* 分组框样式 */
        QGroupBox {
            border: 1px solid #e0e0e0;
            border-radius: 4px;
            margin-top: 10px;
            padding: 8px;
        }

        QGroupBox::title {
            subcontrol-origin: margin;
            left: 10px;
            padding: 0 5px;
            color: #666666;
        }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;设置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>重设密码</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>当前密码</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>新的密码</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>再次输入</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLineEdit" name="nowPasswordEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="newPasswordEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="againEdit"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>确定修改</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>设置背景</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>选择背景图片</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="backgroundEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="backgroundButton">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>设置字体</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>字体</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFontComboBox" name="fontComboBox"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>设置刷新时间</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>刷新时间</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>单位为秒</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
